R/dataprep_table_formatting.R

```diff
@@ -76,7 +76,7 @@ fix_logical_update_existing <- function(sidora_table){
   
   if ("sample.Ethically_culturally_sensitive" %in% names(sidora_table)) {
     result <- sidora_table %>% 
-      dplyr::mutate(dplyr::across(dplyr::contains(c("sample", ".Ethically_culturally_sensitive")), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
+      dplyr::mutate(dplyr::across(dplyr::contains(c("sample.Ethically_culturally_sensitive")), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
```

Comment:
Why was that separated in the first place? With `c("sample", ".Ethically_culturally_sensitive")`, every column of TAB_Sample was already transformed to character here.

Reply:
Not sure now... indeed doesn't make sense.